fusillade/api/groups/__init__.py

```diff
@@ -59,7 +59,7 @@ def put_groups_roles(token_info: dict, group_id: str):
         group.add_roles(request.json['roles'])
     elif action == 'remove':
         group.remove_roles(request.json['roles'])
-    return make_response('', 200)
+    return make_response(f"Roles: {request.json['roles']} added to {group_id} policy modified.", 200)
```

Comment:
Where is `request` instantiated?

Reply:
`from flask import request` contains information about the current request.